src/aead/chacha20_poly1305.rs

```diff
@@ -278,6 +279,67 @@ mod tests {
                     (CRYPTO_POLY1305_STATE_LEN + 255) / 256);
     }
 
+    #[test]
+    pub fn chacha20_tests(){
+        test::from_file("src/chacha_tests.txt", |section, test_case| {
+            assert_eq!(section, "");
+
+            let key_bytes = test_case.consume_bytes("Key");
+            let mut key = [0u32; super::CHACHA20_KEY_LEN/4];
+            for ki in 0..super::CHACHA20_KEY_LEN/4 {
```

Comment:
Use `use super::*` or whatever above, and don't qualify this with `super::`.

Use spaces around the `/` operator and parentheses around the expression: `(CHACHA20_KEY_LEN / 4)`.